test/unit/component/parser/json/BpmnJsonParser.event.test.ts

```diff
@@ -152,7 +152,7 @@ function executeEventCommonTests(
       });
 
       it(`should NOT convert, when there are several '${eventDefinitionKind}EventDefinition' in the same element${specificTitle}, ${titleForEventDefinitionIsAttributeOf}`, () => {
-        const eventDefinition = [{}, {}];
+        const eventDefinition = ['', {}];
```

Comment:
Normally, it was in the previous PR, but I think there was a problem in the rebase 😕